ansible/roles/bash_alias/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+  - name: Add command aliases
+    lineinfile:
+      path: /home/vagrant/.bash_aliases
+      owner: root
+      state: present
+      create: yes
+      line: "alias {{item.name}}='{{item.command}}'"
+    with_items:
+      - { name: 'syncmaster', command: 'git checkout master; git fetch --all; git merge --ff-only upstream/master; git push; git checkout -'}
+      - { name: 'pr', command: 'git push origin HEAD -u && hub pull-request'}
```

Comment:
Missed this. I reckon we should have a dependency between this role and the hub role in a future change

Reply:
Yes, we need a full review of dependencies at some point.